lib/sql-query-builder/where/translate-sql-where.js

```diff
@@ -109,7 +109,8 @@ function handleValue (node, options) {
   }
 
   if (typeof value === 'string') {
-    value = `'${value}'`
+    // single quotes in the value will break alasql queries. They must be escaped with ''.
+    value = `'${value.replace('\'', '\'\'')}'`
```

Comment:
Must replace all

Putting this into `parseWhereToTree()` is better for code organization.

Reply:
That could work, but it would mean traversing the whole tree twice. The first time in `parseWhereToTree` - because in order to find value nodes to do the `'` replacement, we need to crawl the tree.  We are already crawling the the tree after  `parseWhereToTree` and making string modifications (e.g., [here](https://github.com/koopjs/winnow/blob/master/lib/sql-query-builder/where/translate-sql-where.js#L83)), so I don't think leaving it as is diverges from the existing pattern.

Reply:
Made replace all in 679c81b.